src/main/java/de/tum/in/www1/artemis/repository/ModelingExerciseRepository.java

```diff
@@ -97,6 +97,24 @@ default ModelingExercise findByIdElseThrow(long exerciseId) {
         return findById(exerciseId).orElseThrow(() -> new EntityNotFoundException("Modeling Exercise", exerciseId));
     }
 
+    /**
```

Comment:
I feel like this comment belongs to the method below and not the query, as it describes the behaviour of the method (specifically, throwing if not found).